tests/lwip/Makefile

```diff
@@ -1,4 +1,5 @@
 APPLICATION = lwip
+include ../Makefile.tests_common
```

Comment:
Yes, but for those where including the common Makefile is not possible, I suggest https://github.com/RIOT-OS/RIOT/pull/6444#issuecomment-274091472.